pom.xml

```diff
@@ -33,6 +33,11 @@
             <artifactId>appengine-api-1.0-sdk</artifactId>
             <version>1.9.59</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-language</artifactId>
+            <version>1.55.0</version>
+        </dependency>
```

Comment:
I couldn't run this code, because compiled didn't see junit library, add dependency for it